src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -626,6 +623,10 @@
   <ItemGroup>
     <Compile Include="src\System\PinnableBufferCacheEventSource.cs" />
     <Compile Include="src\System\Runtime\RuntimeImports.cs" />
+    <Compile Include="shared\System\Collections\Concurrent\ConcurrentQueue.cs" />
+    <Compile Include="shared\System\Collections\Concurrent\ConcurrentQueue_Segment.cs" />
+    <Compile Include="shared\System\Collections\Concurrent\IProducerConsumerCollection.cs" />
+    <Compile Include="shared\System\Collections\Concurrent\ProducerConsumerCollectionDebugView.cs" />
```

Comment:
I had to add these files to the csproj to fix compile issues with the project. Please let me know if this is OK.

Reply:
I think you are still going to have compile issues.

The shared files should not be included from this project file.

I think the problem is that you have these files in Windows specific section in the shared project file. Try moving them to the platform neutral section.